analyzers/src/SonarAnalyzer.CSharp.Styling/SonarAnalyzer.CSharp.Styling.csproj

```diff
@@ -27,4 +29,19 @@
     <Using Include="Microsoft.CodeAnalysis.Diagnostics" />
   </ItemGroup>
 
+  <Target Name="CopyBinaries" AfterTargets="Build">
+    <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.dll" />
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.CSharp.dll" />
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.CFG.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BinariesToCopy)" DestinationFolder="$(BinariesFolder)" />
+    <!-- Avoid this analyzer to picked up by Java ITs by putting it outside of the binaries folder -->
```

Comment:
This should be in the `Directory.Build.Targets` file, because that's where the original mention about ITs is too.